clients/cloud/kafkacat/kafkacat-example.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -eu
+
+# Set topic name
+topic_name=test3
+
+# Create topic in Confluent Cloud
+ccloud topic create $topic_name || true
+
+# Produce messages
+num_messages=10
+(for i in `seq 0 $num_messages`; do echo "alice,{\"count\":${i}}" ; done) | \
```

Comment:
seq 0 10 gives 11 messages, so should probably start at 1

Reply:
Ack.  Also fixed in confluent CLI